.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: java
-
 jdk:
-  - oraclejdk8
+- oraclejdk8
+after_success:
+- test "${TRAVIS_PULL_REQUEST}" == "false" && test "${TRAVIS_TAG}" != "" && BUILD_NUMBER=$TRAVIS_TAG
```

Comment:
Can simplify into this, I think
`test "$TRAVIS_PULL_REQUEST" == false && test -n "$TRAVIS_TAG" && BUILD_NUMBER=$TRAVIS_TAG`